collection-scripts/gather_virtualmachines

```diff
@@ -1,33 +1,27 @@
-#!/bin/bash -x
+#!/bin/bash
 
-BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
+export BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
 PROS=${PROS:-5}
 
 function collect_vm() {
```

Comment:
this is a misnomer now. It no longer collects a vm, just extracts it from stdin.